frontend/src/apis/index.ts

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+const api = axios.create({
+  baseURL: process.env.API_URL!,
```

Comment:
```suggestion
  baseURL: process.env.API_URL,
```

여기도 마찬가지로 `!` 빼도 될 것 같습니다.